scripts/globals/items/bottle_of_ascetics_tonic.lua

```diff
@@ -0,0 +1,22 @@
+-----------------------------------------
+-- ID: 5841
+-- Item: Bottle of Ascetic's Tonic
+-- Item Effect: Temporarily increases magic atk power and acc
+-----------------------------------------
+require("scripts/globals/status")
+-----------------------------------------
+-- OnItemCheck
+-----------------------------------------
+
+function onItemCheck(target)
+    return 0
+end
+
+-----------------------------------------
+-- OnItemUse
```

Comment:
------------------------------------
-- we don't do that anymore
------------------------------------

Reply:
All gone now